metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/DataProcessAspect.pdl

```diff
@@ -0,0 +1,10 @@
+namespace com.linkedin.metadata.aspect
+
+import com.linkedin.common.Ownership
+import com.linkedin.common.Status
+import com.linkedin.dataprocess.DataProcessInfo
+
+/**
+ * A union of all supported metadata aspects for a data process
+ */
+typeref DataProcessAspect = union[Ownership, Status, DataProcessInfo]
```

Comment:
Is the Status aspect granular enough for a Data Process?

Reply:
For right now, `Status` right now has only `removed` boolean field. I think `Status` aspect should be part of `Data Process`'s aspect. But we need to add more for `Status`, such as `deactivated`

Reply:
In that case, not add status at this point. once you have figured out what you need for status then add it at that point?

Reply:
OK. fixed